tests/test_tables.py

```diff
@@ -6,4 +6,6 @@
 filename = os.path.join(test_dir, 'tables2.html')
 d = HtmlToDocx()
 
+d.table_style = 'Light Grid Accent 6'

```

Comment:
I think it would be helpful to still produce a test output for unspecified `table_style` in addition to the styled version so we can confirm the backwards compatibility is maintained.

Reply:
Yes, good plan. I'll add this in the next few days.

Reply:
I decided to go the other way and revert the `test_tables.py` module to it's original state and instead add the following unittests:

- Insert tables with table grid style
- Insert tables with light accent style
- Insert nested tables with table grid style
- Insert nested tables with light accent style

I'm not sure the `test_tables.py` module is actually needed - it would be better to move it to a unittest.

Reply:
Awesome, thank you. Good idea with adding the cases as unittests. I will look at cleaning up the tests a bit in the next few days.